README.zh-CN.md

```
@@ -15,21 +15,27 @@ npx @codenoobforreal/clitools
 ### 视频编码与压缩
 
 - ​**高效H.265/HEVC转换**​：通过FFmpeg使用[HandBrake推荐质量设置](https://handbrake.fr/docs/en/1.9.0/workflow/adjust-quality.html)批量编码视频至H.265格式
-- 预设参数：当前支持H.265视频编码
+- 预设支持：当前支持H.265视频编码
 
-**环境依赖**​：  
-本功能需要安装 [FFmpeg](https://ffmpeg.org)  
-[安装指南](#ffmpeg-installation-guide)
+- **环境依赖**​：  
+  本功能需要安装 [FFmpeg](https://ffmpeg.org)  
+  [安装指南](#ffmpeg-installation-guide)
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid
null

(MD051, link-fragments)